### PR TITLE
NXP-30608: Update messages.json

### PR DESCRIPTION
**Pull request by jaubenque**
Labels reviewed by Product Management

### CODE REVIEW

**Comment by jaubenque** [comment]
Can you please backport those changes on 10.10 once validated ?

**Comment by guirenard** [comment]
@jaubenque , @manonlumeau , shouldn't @carolinagouveia be reviewing this too?

**Review by guirenard — DISMISSED** [review]

File: nuxeo-coldstorage-web/i18n/messages.json [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
-  "documentContentView.retrieveFromColdStorage.description": "This will result in an asynchronous call and you’ll get an email once the document is ready.",
-  "documentContentView.retrieveFromColdStorage.label": "Retrieve from cold storage",
-  "documentContentView.retrieveFromColdStorage.tooltip": "Retrieve from cold storage",
-  "documentContentView.retrieveFromColdStorage.success": "Start retrieving document content from cold storage",
-  "documentContentView.retrieveFromColdStorage.error": "Error retrieving the document content from cold storage.",
+  "documentContentView.retrieveFromColdStorage.ask.confirmation": "Retrieve this file from cold storage?",
+  "documentContentView.retrieveFromColdStorage.description": "This will result in an asynchronous action and you will receive an email once the file is ready.",
+  "documentContentView.retrieveFromColdStorage.label": "Retrieve file from cold storage",
+  "documentContentView.retrieveFromColdStorage.tooltip": "Retrieve file from cold storage",
+  "documentContentView.retrieveFromColdStorage.success": "Begin restoring the file from cold storage.",
```
Comment (manonlumeau) [inline nuxeo-coldstorage-web/i18n/messages.json]:
```suggestion
  "documentContentView.retrieveFromColdStorage.success": "File restoration from cold storage has started.",
```

File: nuxeo-coldstorage-web/i18n/messages.json [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
-  "documentContentView.restoreFromColdStorage.description": "This will undo the send to cold storage action on the document.",
-  "documentContentView.restoreFromColdStorage.label": "Restore from cold storage",
-  "documentContentView.restoreFromColdStorage.tooltip": "Restore from cold storage",
-  "documentContentView.restoreFromColdStorage.success": "Start restoring the document content from cold storage",
-  "documentContentView.restoreFromColdStorage.error": "Error restoring the document content from cold storage.",
+  "documentContentView.restoreFromColdStorage.ask.confirmation": "Restore the file from cold storage?",
+  "documentContentView.restoreFromColdStorage.description": "This will undo the "Send to cold storage" action on the file.",
+  "documentContentView.restoreFromColdStorage.label": "Restore file from cold storage.",
+  "documentContentView.restoreFromColdStorage.tooltip": "Restore file from cold storage.",
+  "documentContentView.restoreFromColdStorage.success": "Begin restoring the file from cold storage.",
```
Comment (manonlumeau) [inline nuxeo-coldstorage-web/i18n/messages.json]:
```suggestion
  "documentContentView.restoreFromColdStorage.success": "File restoration from cold storage has started.",
```

**Review by manonlumeau — APPROVED** [review]

**Review by guirenard — APPROVED** [review]

**Comment by kevinleturc** [comment]
Merged into #131

**Pull request closed without merging**